--- webrtc/src/api/video_codecs/sdp_video_format.h
+++ webrtc/src/api/video_codecs/sdp_video_format.h
@@ -66,7 +66,7 @@ struct RTC_EXPORT SdpVideoFormat {
 // SdpVideoFormat, which makes instances that should compare equal to not match
 // anymore. Until we stop misusing SdpVideoFormats provide this convenience
 // function to perform fuzzy matching.
-absl::optional<SdpVideoFormat> FuzzyMatchSdpVideoFormat(
+RTC_EXPORT absl::optional<SdpVideoFormat> FuzzyMatchSdpVideoFormat(
     rtc::ArrayView<const SdpVideoFormat> supported_formats,
     const SdpVideoFormat& format);
