diff --git api/audio_codecs/builtin_audio_encoder_factory.h api/audio_codecs/builtin_audio_encoder_factory.h
index 8c0a4ab..0a81db9 100644
--- api/audio_codecs/builtin_audio_encoder_factory.h
+++ api/audio_codecs/builtin_audio_encoder_factory.h
@@ -21,7 +21,7 @@ namespace webrtc {
 // only need a subset of the codecs, consider using
 // CreateAudioEncoderFactory<...codecs listed here...>() or
 // CreateOpusAudioEncoderFactory() instead.
-scoped_refptr<AudioEncoderFactory> CreateBuiltinAudioEncoderFactory();
+RTC_EXPORT scoped_refptr<AudioEncoderFactory> CreateBuiltinAudioEncoderFactory();
 
 }  // namespace webrtc
 
