diff --git modules/video_coding/svc/scalability_mode_util.h modules/video_coding/svc/scalability_mode_util.h
index 9c8193e037..6f346e50c6 100644
--- modules/video_coding/svc/scalability_mode_util.h
+++ modules/video_coding/svc/scalability_mode_util.h
@@ -25,7 +25,7 @@ enum class ScalabilityModeResolutionRatio {
 
 static constexpr char kDefaultScalabilityModeStr[] = "L1T2";
 
-absl::optional<ScalabilityMode> ScalabilityModeFromString(
+RTC_EXPORT absl::optional<ScalabilityMode> ScalabilityModeFromString(
     absl::string_view scalability_mode_string);
 
 InterLayerPredMode ScalabilityModeToInterLayerPredMode(
