diff --git modules/video_coding/codecs/av1/dav1d_decoder.h modules/video_coding/codecs/av1/dav1d_decoder.h
index c9396d1e03..a08febec3a 100644
--- modules/video_coding/codecs/av1/dav1d_decoder.h
+++ modules/video_coding/codecs/av1/dav1d_decoder.h
@@ -16,7 +16,7 @@
 
 namespace webrtc {
 
-std::unique_ptr<VideoDecoder> CreateDav1dDecoder();
+RTC_EXPORT std::unique_ptr<VideoDecoder> CreateDav1dDecoder();
 
 }  // namespace webrtc
 
