diff --git modules/video_coding/codecs/av1/av1_svc_config.h modules/video_coding/codecs/av1/av1_svc_config.h
index 05b886b9f4..7bd207d80a 100644
--- modules/video_coding/codecs/av1/av1_svc_config.h
+++ modules/video_coding/codecs/av1/av1_svc_config.h
@@ -17,10 +17,10 @@
 
 namespace webrtc {
 
-absl::InlinedVector<ScalabilityMode, kScalabilityModeCount>
+RTC_EXPORT absl::InlinedVector<ScalabilityMode, kScalabilityModeCount>
 LibaomAv1EncoderSupportedScalabilityModes();
 
-bool LibaomAv1EncoderSupportsScalabilityMode(ScalabilityMode scalability_mode);
+RTC_EXPORT bool LibaomAv1EncoderSupportsScalabilityMode(ScalabilityMode scalability_mode);
 
 // Fills `video_codec.spatialLayers` using other members.
 bool SetAv1SvcConfig(VideoCodec& video_codec,
