--- webrtc/src/sdk/BUILD.gn
+++ webrtc/src/sdk/BUILD.gn
@@ -135,6 +135,12 @@ if (is_ios || is_mac) {
   }

   rtc_library("helpers_objc") {
+    visibility = [ "*" ]
+
+    public = [
+      "objc/helpers/AVCaptureSession+DevicePosition.h",
+    ]
+
     sources = [
       "objc/helpers/AVCaptureSession+DevicePosition.h",
       "objc/helpers/AVCaptureSession+DevicePosition.mm",
@@ -1286,6 +1292,7 @@ if (is_ios || is_mac) {
           "objc/components/video_codec/RTCVideoEncoderFactoryH264.h",
           "objc/components/video_codec/RTCVideoEncoderH264.h",
           "objc/components/video_frame_buffer/RTCCVPixelBuffer.h",
+          "objc/helpers/AVCaptureSession+DevicePosition.h",
           "objc/helpers/RTCCameraPreviewView.h",
           "objc/helpers/RTCDispatcher.h",
           "objc/helpers/UIDevice+RTCDevice.h",
@@ -1334,6 +1341,7 @@ if (is_ios || is_mac) {
           "objc/api/video_codec/RTCVideoEncoderAV1.h",
           "objc/api/video_frame_buffer/RTCNativeI420Buffer.h",
           "objc/api/video_frame_buffer/RTCNativeMutableI420Buffer.h",
+          "../rtc_base/system/gcd_helpers.h",
         ]

         if (!build_with_chromium) {
@@ -1352,7 +1360,43 @@ if (is_ios || is_mac) {
           "@rpath/$output_name.framework/$output_name",
         ]

+        public_deps = [
+          "../third_party/abseil-cpp/absl/types:bad_optional_access",
+          "../rtc_base/system:gcd_helpers",
+          ":helpers_objc",
+        ]
+
         deps = [
+          "../api:create_peerconnection_factory",
+          "../api:libjingle_peerconnection_api",
+          "../api:rtc_error",
+          "../api:transport_api",
+          "../api/crypto",
+          "../api/rtc_event_log:rtc_event_log_factory",
+          "../api/task_queue",
+          "../api/task_queue:default_task_queue_factory",
+          "../api/video_codecs:builtin_video_decoder_factory",
+          "../api/video_codecs:builtin_video_encoder_factory",
+          "../api/audio_codecs:builtin_audio_decoder_factory",
+          "../api/audio_codecs:builtin_audio_encoder_factory",
+          "../audio",
+          "../call",
+          "../common_audio",
+          "../common_video",
+          "../logging:rtc_event_log_api",
+          "../media",
+          "../modules",
+          "../modules/video_capture:video_capture_internal_impl",
+          "../p2p:rtc_p2p",
+          "../pc:libjingle_peerconnection",
+          "../pc:peerconnection",
+          "../pc:rtc_pc",
+          "../video",
+          "../api",
+          "../logging",
+          "../p2p",
+          "../pc",
+          "../stats",
           ":audio_objc",
           ":base_objc",
           ":default_codec_factory_objc",
