diff --git sdk/objc/native/api/video_encoder_factory.h sdk/objc/native/api/video_encoder_factory.h
index ecd9ab090b..b735d07913 100644
--- sdk/objc/native/api/video_encoder_factory.h
+++ sdk/objc/native/api/video_encoder_factory.h
@@ -18,7 +18,7 @@
 
 namespace webrtc {
 
-std::unique_ptr<VideoEncoderFactory> ObjCToNativeVideoEncoderFactory(
+RTC_EXPORT std::unique_ptr<VideoEncoderFactory> ObjCToNativeVideoEncoderFactory(
     id<RTC_OBJC_TYPE(RTCVideoEncoderFactory)> objc_video_encoder_factory);
 
 }  // namespace webrtc
