diff --git sdk/objc/native/src/objc_video_encoder_factory.h sdk/objc/native/src/objc_video_encoder_factory.h
index 85a1e5319d..458604d0f6 100644
--- sdk/objc/native/src/objc_video_encoder_factory.h
+++ sdk/objc/native/src/objc_video_encoder_factory.h
@@ -24,7 +24,7 @@ namespace webrtc {
 
 class ObjCVideoEncoderFactory : public VideoEncoderFactory {
  public:
-  explicit ObjCVideoEncoderFactory(id<RTC_OBJC_TYPE(RTCVideoEncoderFactory)>);
+  RTC_EXPORT explicit ObjCVideoEncoderFactory(id<RTC_OBJC_TYPE(RTCVideoEncoderFactory)>);
   ~ObjCVideoEncoderFactory() override;
 
   id<RTC_OBJC_TYPE(RTCVideoEncoderFactory)> wrapped_encoder_factory() const;
