diff --git sdk/objc/api/peerconnection/RTCPeerConnectionFactoryBuilder.h sdk/objc/api/peerconnection/RTCPeerConnectionFactoryBuilder.h
index 81d9a3f..b7a843b 100644
--- sdk/objc/api/peerconnection/RTCPeerConnectionFactoryBuilder.h
+++ sdk/objc/api/peerconnection/RTCPeerConnectionFactoryBuilder.h
@@ -22,6 +22,7 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+RTC_OBJC_EXPORT
 @interface RTCPeerConnectionFactoryBuilder : NSObject
 
 + (RTCPeerConnectionFactoryBuilder *)builder;
