diff --git sdk/objc/native/api/video_decoder_factory.h sdk/objc/native/api/video_decoder_factory.h
index 9ba11d65a3..1636ffa0cd 100644
--- sdk/objc/native/api/video_decoder_factory.h
+++ sdk/objc/native/api/video_decoder_factory.h
@@ -18,7 +18,7 @@
 
 namespace webrtc {
 
-std::unique_ptr<VideoDecoderFactory> ObjCToNativeVideoDecoderFactory(
+RTC_EXPORT std::unique_ptr<VideoDecoderFactory> ObjCToNativeVideoDecoderFactory(
     id<RTC_OBJC_TYPE(RTCVideoDecoderFactory)> objc_video_decoder_factory);
 
 }  // namespace webrtc
