--- webrtc/src/sdk/objc/native/api/video_decoder_factory.h
+++ webrtc/src/sdk/objc/native/api/video_decoder_factory.h
@@ -19,7 +19,7 @@
 
 namespace webrtc {
 
-std::unique_ptr<VideoDecoderFactory> ObjCToNativeVideoDecoderFactory(
+RTC_EXPORT std::unique_ptr<VideoDecoderFactory> ObjCToNativeVideoDecoderFactory(
     id<RTC_OBJC_TYPE(RTCVideoDecoderFactory)> objc_video_decoder_factory);
 
 }  // namespace webrtc
