diff --git sdk/objc/native/src/objc_video_decoder_factory.h sdk/objc/native/src/objc_video_decoder_factory.h
index 19c997e503..0b999696a5 100644
--- sdk/objc/native/src/objc_video_decoder_factory.h
+++ sdk/objc/native/src/objc_video_decoder_factory.h
@@ -23,7 +23,7 @@ namespace webrtc {
 
 class ObjCVideoDecoderFactory : public VideoDecoderFactory {
  public:
-  explicit ObjCVideoDecoderFactory(id<RTC_OBJC_TYPE(RTCVideoDecoderFactory)>);
+  RTC_EXPORT explicit ObjCVideoDecoderFactory(id<RTC_OBJC_TYPE(RTCVideoDecoderFactory)>);
   ~ObjCVideoDecoderFactory() override;
 
   id<RTC_OBJC_TYPE(RTCVideoDecoderFactory)> wrapped_decoder_factory() const;
